build_variables.bzl

```diff
@@ -894,6 +894,7 @@ libtorch_python_core_sources = [
     "torch/csrc/autograd/python_torch_functions_manual.cpp",
     "torch/csrc/autograd/python_variable.cpp",
     "torch/csrc/autograd/python_variable_indexing.cpp",
+    "torch/csrc/functorch/CompileCache.cpp",
```

Comment:
Unfortunately there is no way to glob everything under torch/csrc/functorch. I tried a couple of things and it looks like everyone else (see the rest of the files listed here) gave up on this as well